Assets/_Application/Layers/LayerTypes/GeoJsonLayers/GeoJSONPolygonLayer.cs

```diff
@@ -149,40 +150,73 @@ public void AddAndVisualizeFeature<T>(Feature feature, CoordinateSystem original
                     newFeatureVisualisation.AppendVisualisations(GeometryVisualizationFactory.CreatePolygonVisualization(
                         multiPolygon, 
                         originalCoordinateSystem, 
-                        featureRenderMaterial
+                        defaultMaterial
```

Comment:
dit maakt uiteindelijk een PolygonVisualisation, en daar staat dit in de OnDestroy:
`// VisualisationMaterial is global (for now), so they don't have to be destroyed here`
maar nu is dat dus niet meer global, dus ook voor de memory leaks moet er denk ik iets bij om dit weer op te ruimen

Reply:
Is aangepakt in een herbruikbare MaterialApplicator klasse